### PR TITLE
[Task#68809] Modify welcome page and fix localization

### PR DESCRIPTION
**Pull request by tungbd-0214**
## Related Tickets:
- [#68809D](https://edu-redmine.sun-asterisk.vn/issues/68809)

## WHAT this PR does?
Modify welcome page and fix localization

## Notes Impacted Areas:
- N/A.

## Attachments:
Screen Shot 2023-09-16 at 23 24 46

Screen Shot 2023-09-16 at 22 46 07

### CODE REVIEW

**Comment by tungbd-0214** [comment]
ready

File: resources/js/Pages/Welcome.jsx [hunk 1]
```diff
@@ -6,207 +6,70 @@ export default function Welcome(props) {
...
-                            </g>
-                        </svg>
+                    <div className="text-center justify-center pt-8 sm:justify-start sm:pt-0">
+                        <p className="text-4xl text-black dark:text-slate-50">Welcome to best hair salon management system in the world</p>
```
Comment (ghost) [inline resources/js/Pages/Welcome.jsx]: localization

File: resources/js/Pages/Welcome.jsx [hunk 2]
```diff
@@ -6,207 +6,70 @@ export default function Welcome(props) {
...
-                                        development skills in the process.
+                            {props.auth.user ? (
+                                <div className="w-full p-6 border-t border-gray-200 dark:border-gray-700 md:border-t-0 md:border-l">
+                                        <div className="text-center">
+                                            <div className="ml-4 text-2xl leading-7 font-semibold">
+                                                <a
+                                                    href={route('dashboard')}
+                                                    className="underline text-gray-900 dark:text-white"
+                                                >
+                                                    View My Salon's Dashboard
```
Comment (ghost) [inline resources/js/Pages/Welcome.jsx]: localization

File: resources/js/Pages/Welcome.jsx [hunk 3]
```diff
@@ -6,207 +6,70 @@ export default function Welcome(props) {
...
+                                                    className="underline text-gray-900 dark:text-white"
+                                                >
+                                                    Login
+                                                </a>
+                                            </div>
+                                        </div>
+
+                                        <div className="ml-12">
+                                            <div className="mt-2 text-gray-600 dark:text-gray-400 text-sm">
+                                                Laracasts offers thousands of video tutorials on Laravel, PHP, and JavaScript
```
Comment (ghost) [inline resources/js/Pages/Welcome.jsx]: localization

File: resources/js/Pages/Welcome.jsx [hunk 4]
```diff
@@ -6,207 +6,70 @@ export default function Welcome(props) {
...
-                                    <div className="ml-4 text-lg leading-7 font-semibold text-gray-900 dark:text-white">
-                                        Vibrant Ecosystem
+                                    <div className="p-6 border-t border-gray-200 dark:border-gray-700">
+                                        <div className="text-center">
+                                            <div className="ml-4 text-2xl leading-7 font-semibold">
+                                                <a
+                                                    href={route('register')}
+                                                    className="underline text-gray-900 dark:text-white"
+                                                >
+                                                    Register
```
Comment (ghost) [inline resources/js/Pages/Welcome.jsx]: localization

File: resources/js/Pages/Welcome.jsx [hunk 5]
```diff
@@ -6,207 +6,70 @@ export default function Welcome(props) {
...
+                                                </a>
+                                            </div>
+                                        </div>
+
+                                        <div className="ml-12">
+                                            <div className="mt-2 text-gray-600 dark:text-gray-400 text-sm">
+                                                Before efficently manage your salon, you have to register your salon in our system and we will provide all you need to make all things becomes easy !!
```
Comment (ghost) [inline resources/js/Pages/Welcome.jsx]: localization

File: resources/js/Pages/Welcome.jsx [hunk 6]
```diff
@@ -247,7 +110,7 @@ export default function Welcome(props) {
                         </div>
 
                         <div className="ml-4 text-center text-sm text-gray-500 sm:text-right sm:ml-0">
-                            Laravel v{props.laravelVersion} (PHP v{props.phpVersion})
+                            Power by Hedspi Team
```
Comment (ghost) [inline resources/js/Pages/Welcome.jsx]: localization

**Comment by tungbd-0214** [comment]
ready

**Pull request merged**